src/olympia/stats/views.py

```diff
@@ -297,12 +297,21 @@ def check_series_params_or_404(group, start, end, format):
     return get_daterange_or_404(start, end)
 
 
-def check_stats_permission(request, addon):
+def check_stats_permission(request, addon, beta):
```

Comment:
Personal preference: I don't like booleans as positional arguments - If someone ends up calling the function with just `True` or `False` passed as the value, like in `foo(True)`, it makes it difficult to understand what the boolean represents at first glance. So I always try to remember to set them as keyword (or even keyword-only) arguments. 

It matters slightly less in this instance as you did directly pass the `beta` variable in every call, which is explicit enough, and we're unlikely to have many more calls to that function, so you can ignore that comment for this PR.